pm/sendermonitor_test.go

```diff
@@ -380,6 +424,51 @@ func TestCleanup(t *testing.T) {
 	mf2, err = sm.MaxFloat(addr2)
 	require.Nil(err)
 
-	assert.Equal(reserve4, mf1)
-	assert.Equal(reserve5, mf2)
+	expectedAlloc4 := new(big.Int).Sub(new(big.Int).Div(smgr.info[addr2].Reserve, rm.transcoderPoolSize), smgr.claimedReserve[addr2])
+	assert.Equal(expectedAlloc3, mf1)
+	assert.Equal(expectedAlloc4, mf2)
+}
+
+func TestReserveAlloc(t *testing.T) {
+	assert := assert.New(t)
+
+	claimant := RandAddress()
+	b := newStubBroker()
+	addr := RandAddress()
+	smgr := newStubSenderManager()
+	smgr.info[addr] = &SenderInfo{
```

Comment:
Looks like this initialization code is used a lot in this file. Could we extract it into a helper?